Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -28,7 +28,7 @@ namespace UnityEngine.InputSystem.Editor
     [ScriptedImporter(kVersion, InputActionAsset.Extension)]
     internal class InputActionImporter : ScriptedImporter
     {
-        private const int kVersion = 13;
+        internal const int kVersion = 14;
```

Comment:
Makes sense to bump this one as you have done to trigger re-import to ensure parameters have been correctly parsed, see https://docs.unity3d.com/ScriptReference/AssetImporters.ScriptedImporterAttribute-ctor.html

Reply:
I think it should be private since uncorrelated to JSON version. Sorry for saying there should be only one earlier. That was incorrect.